media/css/mozorg/diversity/2021/diversity-2021.scss

```diff
@@ -0,0 +1,475 @@
+    padding-top: $spacing-2xl;
+    color: $color-black;
+
+    a:link,
+    a:visited {
+        color: $color-black;
+
+        &:hover,
+        &:focus,
+        &:active {
+            color: get-theme('link-color-hover');
+        }
+    }
+
+    .mzp-c-card-feature-content-container {
+        position: relative;
+
+        img {
+            margin-top: -$spacing-2xl;
+            margin-bottom: $spacing-md;
+            width: 100%;
+            max-width: $content-sm;
+            height: auto;
+            display: none;
+
+            @media #{$mq-md} {
+                display: block;
+            }
+
+            @media #{$mq-lg} {
+                margin-top: -$layout-lg;
+            }
+        }
+
+        p {
+            margin-bottom: $spacing-md;
+        }
+
+        ul,
+        ol {
+            @include text-body-md;
+            color: $color-marketing-gray-90;
+        }
+    }
+}
+
+.mzp-c-modal-close {
+    background: $color-green-40;
+    border-radius: 100%;
+
+    button {
+        cursor: pointer;
+    }
+
+    .mzp-c-modal-button-close {
+        background-image: url('/media/protocol/img/icons/close.svg');
+    }
+}
+
+.c-modal-next {
+    @extend .mzp-c-modal-close; /* stylelint-disable-line scss/at-extend-no-missing-placeholder  */
+    right: 3.75rem;
+
+    .hide-next & {
+        display: none;
+    }
+
+    .c-modal-button-next {
+        background-image: url('/media/protocol/img/icons/arrow-right.svg');
+    }
+}
+
+.c-modal-prev {
+    @extend .mzp-c-modal-close; /* stylelint-disable-line scss/at-extend-no-missing-placeholder  */
+    right: 7.1rem;
+
+    .hide-next & {
+        right: 3.75rem;
+    }
+
+    .hide-prev & {
+        display: none;
+    }
+
+    .c-modal-button-prev {
+        background-image: url('/media/protocol/img/icons/arrow-left.svg');
+    }
+}
+
+.c-modal-button-next,
+.c-modal-button-prev {
+    @extend .mzp-c-modal-button-close; /* stylelint-disable-line scss/at-extend-no-missing-placeholder */
+}
+
+.c-section-intro-bg {
+    @media #{$mq-lg} {
+        background: url('/media/img/foundation/annualreport/2019/moz_site_som_fighting_for_people_left.svg') left center no-repeat, url('/media/img/foundation/annualreport/2019/moz_site_som_fighting_for_people_right.svg') right center no-repeat;
+        background-size: auto 100%;
+    }
+}
+
+.mzp-l-content + .c-section-intro-bg {
+    margin-top: $spacing-2xl;
+}
+
+.c-section-intro {
+    margin-top: $layout-xl;
+}
+
+.c-section-wrapper {
+    margin: 0 auto;
+    padding-top: $layout-sm;
+    max-width: $content-lg;
+
+    h2 {
+        @include text-title-lg;
+        font-weight: 100;
+        text-align: center;
+    }
+
+    h3 {
+        @include text-title-sm;
+        font-weight: 600;
+        text-align: left;
+    }
+
+    h4 {
+        @include text-title-xs;
+        font-weight: 600;
+        text-align: left;
+    }
+
+    .c-subtitle {
+        text-align: center;
+    }
+
+    ul {
```

Comment:
It looks like these are all `mzp-u-list-style` already so I think these styles are redundant.